apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/index.tsx

```diff
@@ -49,6 +53,31 @@ export const LargeMoverLandingPage = ({ route }: LargeMoverLandingPageProps) =>
   const { colors } = useTheme();
   const height = getWindowDimensions().height * 0.75;
   const [currentIndex, setCurrentIndex] = useState(0);
+  const dispatch = useDispatch();
+  const showOverlay = useSelector(tutorialSelector);
+  const handleSwipe = (newIndex: number) => {
+    const from = currenciesWithId[currentIndex].data?.name;
+    const to = currenciesWithId[newIndex].data?.name;
+
+    track("large_mover_swipe", {
+      page: PAGE_NAME,
+      button: "Swipe",
+      from,
+      to,
+    });
+
+    setCurrentIndex(newIndex);
+  };
+
+  useEffect(() => {
+    getLargeMoverState().then(state => {
+      if (state && typeof state.tutorial === "boolean") {
+        dispatch(setTutorial(state.tutorial));
+      } else {
+        dispatch(setTutorial(true));
+      }
+    });
+  }, [dispatch]);
```

Comment:
I'm a bit confused - doesn't LedgerStore.tsx handle this? 

I _think_ if you correct the payload as I suggest, this useEffect becomes redundant. I might've misunderstood though